src/apps/angle/ethereum/angle.angle.token-fetcher.ts

```diff
@@ -0,0 +1,77 @@
+import { Inject } from '@nestjs/common';
+import { utils } from 'ethers';
+
+import { IAppToolkit, APP_TOOLKIT } from '~app-toolkit/app-toolkit.interface';
+import { Register } from '~app-toolkit/decorators';
+import { buildDollarDisplayItem } from '~app-toolkit/helpers/presentation/display-item.present';
+import { getImagesFromToken } from '~app-toolkit/helpers/presentation/image.present';
+import { ContractType } from '~position/contract.interface';
+import { BalanceDisplayMode } from '~position/display.interface';
+import { PositionFetcher } from '~position/position-fetcher.interface';
+import { AppTokenPosition } from '~position/position.interface';
+import { Network } from '~types/network.interface';
+
+import { ANGLE_DEFINITION } from '../angle.definition';
+import { AngleContractFactory } from '../contracts';
+import { fetchTokenList } from '../helpers/angle.api';
+
+const appId = ANGLE_DEFINITION.id;
+const groupId = ANGLE_DEFINITION.groups.angle.id;
+const network = Network.ETHEREUM_MAINNET;
+
+@Register.TokenPositionFetcher({ appId, groupId, network })
+export class EthereumAngleAngleTokenFetcher implements PositionFetcher<AppTokenPosition> {
```

Comment:
Same deal as agEUR, we don't need this token fetcher because we already support ANGLE as a base token: https://zapper.fi/account/0xe02f8e39b8cfa7d3b62307e46077669010883459